### PR TITLE
sc and sc.exe on powershell

### PR DESCRIPTION
**Pull request by OzBob**
updated examples

### CODE REVIEW

**Comment by OzBob** [comment]
fix for #2898

**Comment by illfated** [comment]
Thank you for creating this PR and starting the work to have those examples included. I was thinking about doing the same, more than a month ago, but I kept losing track of what the necessary changes should look like.

**Comment by illfated** [comment]
By the way, maybe we could add a MarkDown Note blob to explain why it is necessary to include the .exe file name extension in the PowerShell command, to highlight that the PowerShell command `sc` is an alias for the command `Set-Content` -- would you like to add that near the PowerShell command (before or after the code example)?

**Comment by illfated** [comment]
@OzBob : are you still following the updates in this PR of yours?

**Comment by illfated** [comment]
(Ref. issue tickets #2898 and #3822 )

**Comment by illfated** [comment]
> **Conflicting files**
> WindowsServerDocs/administration/windows-commands/sc-delete.md

@OzBob : Will you be able to resolve the file merge conflict by modifying the PR content accordingly?

**Comment by illfated** [comment]
See also the latest updates for this file here:
- https://github.com/MicrosoftDocs/windowsserverdocs/commits/master/WindowsServerDocs/administration/windows-commands/sc-delete.md

File: WindowsServerDocs/administration/windows-commands/sc-delete.md [hunk 1]
```diff
@@ -43,11 +43,20 @@ Use **Add or Remove Programs** on **Control Panel** to delete DHCP, DNS, or any
 
 ## Examples
 
+### DOS Examples
```
Comment (skycommand) [inline WindowsServerDocs/administration/windows-commands/sc-delete.md:46]: "DOS" is the name of an operating system that went extinct 25 years ago. I've seen people confusing Command Prompt with DOS, but an official documentation page is not written by confused people.

File: WindowsServerDocs/administration/windows-commands/sc-delete.md [hunk 2]
```diff
@@ -43,11 +43,20 @@ Use **Add or Remove Programs** on **Control Panel** to delete DHCP, DNS, or any
...
 To delete the service subkey **NewServ** from the registry on the local computer, type:
 ```
 sc delete newserv
 ```
 
+### Powershell Examples
+
+To delete the service subkey **NewServ** from the registry on the local computer, type:
+```
+sc.exe delete newserv
```
Comment (skycommand) [inline WindowsServerDocs/administration/windows-commands/sc-delete.md:57]: Simply changing "sc" to "sc.exe" makes the example suitable for both PowerShell and Command Prompt. There is no need for all the bells and whistles of having two separate examples.

**Comment by illfated** [comment]
@OzBob : Are you there? We need to know if you will be able to follow up the feedback provided in this PR. Otherwise, we may as well close this PR and start a new one with an active online user.

**Comment by OzBob** [comment]
@illfated yes changing Sc to Sc.exe makes the example work. Thanks to @skycommand.

**Comment by OzBob** [comment]
I will close this and allow more active users who have better Git skills to resolve the 'update sc to sc.exe' change

**Pull request closed without merging**